packages/pluggableWidgets/radio-buttons-native/src/RadioButtons.tsx

```diff
@@ -0,0 +1,54 @@
+import { createElement, useCallback, ReactElement } from "react";
+import { Text, View } from "react-native";
+
+import { mergeNativeStyles } from "@mendix/pluggable-widgets-tools";
+import { executeAction } from "@mendix/piw-utils-internal";
+
+import { RadioButton } from "./components/RadioButton";
+import { RadioButtonsProps } from "../typings/RadioButtonsProps";
+import { defaultRadioButtonsStyle, RadioButtonsStyle } from "./ui/Styles";
+
+export type props = RadioButtonsProps<RadioButtonsStyle>;
+
+export function RadioButtons({
+    enum: { setValue, universe, value, formatter, readOnly, validation },
+    orientation,
+    style,
+    onChange,
+    name,
+    label,
+    showLabel
+}: props): ReactElement {
+    const styles = mergeNativeStyles(defaultRadioButtonsStyle, style);
+    const onSelect = useCallback(
+        (selectedValue: string) => {
+            if (selectedValue === value || readOnly) {
+                return;
+            }
+            setValue(selectedValue);
+            executeAction(onChange);
+        },
+        [onChange, readOnly, setValue, value]
+    );
+
+    return (
+        <View testID={name}>
+            {showLabel && <Text style={styles.labelTextStyle}>{label?.value}</Text>}
+            <View style={orientation === "horizontal" && styles.containerHorizontal}>
+                {universe?.map(name => (
+                    <RadioButton
+                        key={name}
+                        active={value === name}
+                        onSelect={onSelect}
+                        title={formatter.format(name)}
```

Comment:
`formatter.format(name)` formats `name` using the default formatter. this is the same as using `enum.displayValue`

here's [some documentation](https://docs.mendix.com/apidocs-mxsdk/apidocs/pluggable-widgets-client-apis#editable-value) on EditableValues

Reply:
nevermind, this is right! nice